plugins/inventory/foreman.py

```diff
@@ -65,6 +65,13 @@
             - Places hostvars in a dictionary with keys `foreman`, `foreman_facts`, and `foreman_params`
         type: boolean
         default: False
+      want_ansible_ssh_host:
```

Comment:
I don't know if _this_ parameter is necessary any more since you could add the option

```yaml
compose:
  ansible_ssh_host: foreman['ip6'] | default(foreman['ip'], true)
```

but looking at this now, I'm unclear on if that would handle all the ip, ipv4, ipv6 cases. So maybe this is absolutely necessary.

Reply:
this is also the reasoning @willtome gave in https://github.com/theforeman/foreman-ansible-modules/pull/729#issuecomment-602625977, so I'd *prefer* not to add this, unless there is really a scenario where `compose` does not help us.

Reply:
@evgeni @AlanCoding  I did look at that, but it seems like Satellite has 3 different values that it can populate: ipv6,ipv4,ip. And I am not sure if the compose handles all of that.

Reply:
IMHO this should only be `ip` and `ip6`, at least that's what Foreman provides. Not sure where `ip4` is coming from.

Reply:
@evgeni got it. If that's the case I can may be remove this and let this be handled via compose. On Tower there will be few customers for whom this will be backward incompatible since script had that parameter "want_ansible_ssh_host" and now they will have to change that to compose.

Reply:
While I like a bit of backward compatibility (see also my comment in https://github.com/theforeman/foreman-ansible-modules/pull/799#discussion_r433803264) I don't think it should come at the cost of carrying around code from an old implementation that's not supported anymore.

Reply:
@evgeni looking at this reply here https://github.com/ansible/awx/issues/2916#issuecomment-584834026 looks there was a case when ipv4 was there ?

Reply:
"was there"? :)
The API has `ip` and `ip6`, no `ipv4` and no `ipv6` (note the missing *v* in the name) (see https://github.com/theforeman/foreman/blob/develop/app/models/host/base.rb#L92-L101)

Reply:
@evgeni oh that's interesting. May be we need to check the compose too then. But I have removed this param from this PR. Now this only covers host_filters.
Thanks a lot for looking into it.